.github/workflows/lint.yml

```diff
@@ -27,7 +27,10 @@ jobs:
       - name: Running bazel_to_cmake for IREE TF Integration Tests
         run: ./build_tools/bazel_to_cmake/bazel_to_cmake.py --root_dir=integrations/tensorflow/e2e --verbosity=2
       - name: Checking Diff
-        run: git diff --exit-code
+        run: |
+          # Make sure to pick up new files that have been added.
+          git add -A
```

Comment:
what could possibly go wrong? :P seems fine since this is running on CI..

Reply:
Lol yeah. I also balked at this, but my searches for "diff and also include untracked files" didn't turn up much. I think in other places I use `git status --porcelain`, but that a) uses an API explicitly for human consumption, and b) doesn't actually display the diff.